library/src/main/java/com/xwray/groupie/Item.java

```diff
@@ -40,16 +41,18 @@ public VH createViewHolder(@NonNull View itemView) {
      * @param payloads            Any payloads (this list may be empty)
      * @param onItemClickListener An optional adapter-level click listener
      * @param onItemLongClickListener An optional adapter-level long click listener
+     * @param isSelected
      */
     @CallSuper
     public void bind(@NonNull VH viewHolder, int position, @NonNull List<Object> payloads,
                      @Nullable OnItemClickListener onItemClickListener,
-                     @Nullable OnItemLongClickListener onItemLongClickListener) {
+                     @Nullable OnItemLongClickListener onItemLongClickListener,
+                     boolean isSelected) {
         viewHolder.bind(this, onItemClickListener, onItemLongClickListener);
-        bind(viewHolder, position, payloads);
+        bind(viewHolder, position, payloads, isSelected);
     }
 
-    public abstract void bind(@NonNull VH viewHolder, int position);
+    public abstract void bind(@NonNull VH viewHolder, int position, boolean isSelected);
```

Comment:
adding `isSelected` to the main `bind` function would require everyone who updates to change their code. Instead, create a separate bind function that takes `isSelected` and pass it false from this one.

Reply:
I fixed the code